_includes/headtags.html

```diff
@@ -1,14 +1,11 @@
 <meta charset="utf-8">
 <!-- Canonical and Alternate Links -->
-{% assign sorted_languages = site.languages | sort %}
-{% for lng in sorted_languages %}
-  {% if lng == site.default_lang %}
-    <link rel="canonical" href="{{ site.url }}{{ site.baseurl }}{{ page.url }}">
-  {% else %}
+<link rel="canonical" hreflang="en" href="{{ site.url }}{{ page.url }}">
```

Comment:
By the way, why isn't `{{ page.url }}` here `{{ site.baseurl }}/en/{{ page.url }}`?  It might be worth adding a `{# #}` comment.

Reply:
I wondered that too but not sure what I'd put as a comment since I don't know why it is.

Reply:
Maybe "This works... but keep in view we do not understand why"?

Reply:
There is no /en/ subfolder as far as I know. Also the baseurl would probably just to make it an absolute url

Reply:
`href="{{ site.url }}..."` should be the absolute part.